src/Microsoft.Health.Dicom.Api/Web/IHttpStreamContent.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Health.Dicom.Api.Web;
+
+public interface IHttpStreamContent
```

Comment:
Do we need an interface or is the `Stream` abstraction of the `DicomStreamContent` sufficient (eg. use `MemoryStream` sometimes)? Do you have a scenario in mind where we might want different implementations of the content itself?

Reply:
to keep the Lazystream generic